config/examples/redis.yml

```diff
@@ -3,8 +3,19 @@ base: &default
   size: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
   pool_timeout: 5 # this is in seconds
   sentinels: "<%= ENV['REDIS_SENTINEL_HOSTS'] %>"
+  sentinel_username: <%= ENV['REDIS_SENTINEL_USERNAME'].to_json %>
+  sentinel_password: <%= ENV['REDIS_SENTINEL_PASSWORD'].to_json %>
   name: <%= ENV['REDIS_SENTINEL_NAME'] %>
   role: <%= ENV['REDIS_SENTINEL_ROLE'] %>
+  # == ACL Params ==
+  username: <%= ENV['REDIS_USERNAME'].to_json %>
+  password: <%= ENV['REDIS_PASSWORD'].to_json %>
+  # == TLS Params ==
+  ssl: <%= ENV.fetch('REDIS_SSL', '0') == '1' %>
```

Comment:
In that situation the Redis client will infer `ssl: true`. This variable is optional in almost all cases, it's only mandatory when connecting to TLS protected sentinels.

Reply:
I'm just wondering, when this is set to `false` hopefully it wont enforce no SSL

Reply:
> I'm just wondering, when this is set to `false` hopefully it wont enforce no SSL

I believe it will ONLY enable SSL when `REDIS_SSL=1`, it will be disabled with any other value.

Reply:
> I believe it will ONLY enable SSL when REDIS_SSL=1, it will be disabled with any other value.

True

> I'm just wondering, when this is set to false hopefully it wont enforce no SSL

No, you can ignore the value or set it to `0`, `false` or whatever and it still connects to redis if you provide a certificate. The scenario you mention would only fail for sentinels + TLS, where `REDIS_SSL=1` is mandatory.